activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-api-impl/src/main/java/org/activiti/cloud/modeling/api/impl/ProjectImpl.java

```diff
@@ -62,12 +65,35 @@ public void setId(String id) {
 
     @Override
     public String getName() {
-        return name;
+        return getTechnicalName();
     }
 
     @Override
     public void setName(String name) {
-        this.name = name;
+        setTechnicalName(name);
+
+        //TODO: Modify temporary code that sets display name same as technical name
+        setDisplayName(name);
+    }
```

Comment:
I will leave just a comment since I will be off after today and I don't want to block the PR.
I think we should not set both names here, but just delegate to `setTechnicalName`. If it is required to set also the display name, I think it would be better doing it in another place to avoid having issues when get/set name will be deleted.
In addition, it would be easier implementing `getName` and `setName` with a `default` implementation in the interfaces to avoid repeating the same behavior.